config/images/images.yaml

```diff
@@ -41,12 +41,14 @@ images:
   - v2.0.9
   - v2.0.10
   - v2.1.4
+  - v2.2.0
 - source: kubesphere/fluent-operator
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/kubesphere/fluent-operator
   tags:
   - v1.7.0
   - v2.2.0
   - v2.3.0
+  - v2.7.0
```

Comment:
Did you test that the non-copied versions looks as expected for Gardener?

Reply:
Yes tests are done successfully.